com.regnosys.rosetta/src/com/regnosys/rosetta/generator/java/object/NamespaceHierarchyGenerator.xtend

```diff
@@ -0,0 +1,127 @@
+package com.regnosys.rosetta.generator.java.object
+
+import java.util.Collection
+import java.util.Collections
+import java.util.List
+import java.util.Map
+import java.util.concurrent.atomic.AtomicInteger
+import org.eclipse.xtext.generator.IFileSystemAccess2
+
+class NamespaceHierarchyGenerator {
+
+	val String CDM_NAMESPACE_ROOT = "cdm"
```

Comment:
I dont think we can hard code the model name here.

This will mean that all models will have the root of CDM.